firebase-app-distribution/src/test/java/com/google/firebase/app/distribution/UpdateAppClientTest.java

```diff
@@ -1,144 +0,0 @@
-// Copyright 2021 Google LLC
```

Comment:
A lot of tests were deleted here. Should we move any of them to the FirebaseAppDistributionTest class? I'm particularly interested in preserving the test coverage provided by updateAppTask_whenNoReleaseAvailable_throwsError

Reply:
Added in the tests back in. Decided to keep all of them for now.